e2eshark/requirements.txt

```diff
@@ -1,14 +1,16 @@
-transformers
+tabulate
+simplejson
 ml_dtypes
-huggingface-cli
 onnxruntime
+transformers
+huggingface-cli
 sentencepiece
 accelerate
 auto-gptq
 optimum
-tabulate
+azure-storage-blob
+# install nightly build of torch_mlir 
+torch_mlir -f  https://github.com/llvm/torch-mlir-release/releases/expanded_assets/dev-wheels
 # install nightly build of iree-compiler and iree-runtime
 iree-compiler -f https://iree.dev/pip-release-links.html
 iree-runtime -f https://iree.dev/pip-release-links.html
-azure-storage-blob
-simplejson
```

Comment:
can you add a new line to this file

Reply:
Just add an extra new line to make it easier to read?

Reply:
<img width="646" alt="image" src="https://github.com/nod-ai/SHARK-TestSuite/assets/77521230/055b9527-cb13-4b54-a1dd-f2d599422e88">
This red circle with a line means that we need a new line at the end of the file. Git is able to separate different files using this new line.

Reply:
Got it. Will add.

Reply:
There seems to be new lines now